docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "2"
+services:
+  slackemon_webserver:
+    build: .
+    restart: always
+    volumes: 
+      - ./:/var/www/html
+    ports:
+      - "80:80"
+
+  slackemon_db:
```

Comment:
Docker-compose automatically set the hostname of a machine as the name of that machine. Moreover, it creates a network in which the containers are added